web/src/theme.ts

```diff
@@ -0,0 +1,42 @@
+import { extendTheme } from "@chakra-ui/react";
+
+const customTheme = extendTheme({
+  colors: {
+    primary: "#1E1932",
+    secondary: "#9D9D9D",
+  },
+  styles: {
+    global: {
+      "html, body": {
+        color: "primary", // Définit la couleur par défaut des textes
+      },
+    },
+  },
+  components: {
+    Button: {
+      variants: {
+        1: {
```

Comment:
des noms sont plus parlants

```suggestion
        "primary": {
```